pkg/k8s/utils/utils.go

```diff
@@ -72,9 +72,9 @@ func GetObjNamespaceName(obj NamespaceNameGetter) string {
 
 // ServiceConfiguration is the required configuration for GetServiceListOptionsModifier
 type ServiceConfiguration interface {
-	// K8sServiceProxyName must return the value of the proxy name
+	// K8sServiceProxyNameValue must return the value of the proxy name
```

Comment:
nit: Why is this rename necessary? The new name seems more confusing than the current name.

Reply:
Hey @joestringer. Thanks for the review! Actually, there was a field name `k8sServiceProxyName` which had to be made public for reflections to work. As well there was already a function named `K8sServiceProxyName` to get the value from the struct. So @aanm asked me to change the function name to `K8sServiceProxyNameValue`.

Reply:
Maybe we can rename it to `GetK8sServiceProxyName`.

Reply:
Aha, it was the reflection piece that made it necessary, I see. Reading through I couldn't tell what it conflicted with since the conflicting name isn't touched in this PR at all (maybe because it's already public?).

In this case, I mildly prefer `GetK8sServiceProxyName` but it doesn't make that big of a difference. Don't worry about it :)